basyx/aas/adapter/http.py

```diff
@@ -218,6 +218,22 @@ def is_stripped_request(request: Request) -> bool:
 
 T = TypeVar("T")
 
+ENCODING = "utf-8"
+
+
+def base64url_decode(data: str) -> str:
```

Comment:
We should also add a `base64url_encode` function here and change the `IdentifierConverter` to make use of this function.